src/Services/Web/PrinterGroupWebService.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\Services\Web;
+
+use MyParcelNL\Sdk\Model\Account\PrinterGroup;
+use MyParcelNL\Sdk\Model\MyParcelRequest;
+use MyParcelNL\Sdk\Support\Collection;
+
+class PrinterGroupWebService extends AbstractWebService
+{
+    public const ENDPOINT = 'printer-groups';
+
+    /**
+     * Get all printer groups for the account
+     *
+     * @return \MyParcelNL\Sdk\Support\Collection|\MyParcelNL\Sdk\Model\Account\PrinterGroup[]
+     * @throws \MyParcelNL\Sdk\Exception\AccountNotActiveException
+     * @throws \MyParcelNL\Sdk\Exception\ApiException
+     * @throws \MyParcelNL\Sdk\Exception\MissingFieldException
+     */
+    public function getPrinterGroups(): Collection
+    {
+        $request = $this->createRequest()
+            ->setBaseUrl(MyParcelRequest::PRINTING_API_URL)
+            ->sendRequest('GET', self::ENDPOINT);
+
+        // The API may return printer groups in different structures depending on version
```

Comment:
I do not think this is true, and if it is, I would choose to support the current implementation and not a previous one, since this functionality is new in the SDK anyway.